app/src/main/java/org/xwiki/android/sync/utils/IntentUtils.kt

```diff
@@ -43,6 +49,17 @@ fun openLink(url: String): Intent {
     return intent
 }
 
+fun openLinkInWebView(url: String, webView: WebView)  {
```

Comment:
Could be extension for WebView:

fun WebView.openLink(url: String)

Reply:
Already updated.